### PR TITLE
Add tizen x64 rootfs and cross build support

### PR DESCRIPTION
**Pull request by gbalykov**
Change similar to https://github.com/dotnet/arcade/pull/8609

cc @alpencolt

### CODE REVIEW

**Comment by gbalykov** [comment]
@am11 Thanks, I agree. It's time to unify these scripts.

**Comment by gbalykov** [comment]
I've unified scripts

File: eng/common/cross/tizen-build-rootfs.sh [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+  TIZEN_ARCH="aarch64"
+elif [ "$ARCH" == "x86" ]; then
+  TIZEN_ARCH="i686"
+elif [ "$ARCH" == "x64" ]; then
+  TIZEN_ARCH="x86_64"
+  LINK_ARCH="x86"
+else
+  echo "Unsupported architecture for tizen: $ARCH"
+  exit 1
+fi
```
Comment (am11) [inline eng/common/cross/tizen-build-rootfs.sh:22]: nit: align style with `build-rootfs.sh` (4 spaces indentation, case-esac for switching etc.)

```suggestion
case $ARCH in
    arm)
        TIZEN_ARCH="armv7hl"
        ;;
    armel)
        TIZEN_ARCH="armv7l"
        LINK_ARCH="arm"
        ;;
    arm64)
        TIZEN_ARCH="aarch64"
        ;;
    x86)
        TIZEN_ARCH="i686"
        ;;
    x64)
        TIZEN_ARCH="x86_64"
        LINK_ARCH="x86"
        ;;
    *)
        echo "Unsupported architecture for tizen: $ARCH"
        exit 1
esac
```

File: eng/common/cross/tizen-fetch.sh [hunk 2]
```diff
@@ -51,6 +51,8 @@ if [ ! -d $TMPDIR ]; then
 	mkdir -p $TMPDIR
```
Comment (am11) [inline eng/common/cross/tizen-fetch.sh:51]: nit: 4 space indentation in this file.

**Review by am11 — APPROVED** [review]
Nice cleanup! 👍

**Review by am11 — APPROVED** [review]
LGTM, thank you for the consolidation!
(we can wireup build-rootfs-android the same way some day 💡)

**Review by jkotas — APPROVED** [review]
Thanks

**Comment by gbalykov** [comment]
Thank you

**Pull request merged**